pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -93,8 +93,9 @@ const (
 	addressGroupType   grouping.GroupType = "addressGroup"
 	internalGroupType  grouping.GroupType = "internalGroup"
 
-	perNamespaceRuleIndex = "hasPerNamespaceRule"
-	HasPerNamespaceRule   = "true"
+	perNamespaceRuleIndex   = "hasPerNamespaceRule"
+	namespaceLabelRuleIndex = "namespaceRuleLabelKeys"
+	hasSuchRule             = "true"
```

Comment:
Feels like the name could be mistaken for has the exact same rule, or has rules that match the criteria, etc.

Reply:
Yes, I didn't understand the variable name when looking at its consumer. 
How about `indexValueTrue`

Reply:
Done